src/main/java/monique/task/Deadline.java

```diff
@@ -0,0 +1,126 @@
+package monique.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+import monique.exception.IllegalDateFormatException;
+
+
+/**
+ * The <code>Deadline</code> class represents a task with a specific deadline.
+ * It extends the <code>Task</code> class and includes a due date for the task.
+ */
+public class Deadline extends Task {
+
+    private static final String FORMATSTRING = "[D][%s] %s (by: %s)";
```

Comment:
Would be good to write this in screaming snake case, as FORMAT_STRING! 😄

Reply:
Thanks! I've fixed it :)